sklearn/utils/_random.pxd

```diff
@@ -16,10 +16,6 @@ cdef enum:
     # 32-bit signed integers (i.e. 2^31 - 1).
     RAND_R_MAX = 2147483647
 
-cpdef sample_without_replacement(cnp.int_t n_population,
```

Comment:
This would make `sample_without_replacement` not cimportable by other modules. Given that `_random` is a private module, I am okay with it's removal.